README.md

```diff
@@ -15,14 +15,14 @@ A Gradle plugin to measure the app size impact of Android libraries.
 
 ## Usage
 
-[![Download](https://api.bintray.com/packages/twilio/releases/apkscale/images/download.svg) ](https://bintray.com/twilio/releases/apkscale/_latestVersion)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.twilio/apkscale/badge.svg) ](https://maven-badges.herokuapp.com/maven-central/com.twilio/apkscale)
```

Comment:
Nice! Let's make sure this gets updated after doing the first release to maven central.